woocommerce/woocommerce
# Create conditional-tags.md

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





migrates [conditional tags](https://href.li/?https://woo.com/wp-admin/post.php?post=183&action=edit) document from woo.com



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. single markdown file should only be at risk of failing lint. 
2. check to make sure doc looks ok before merging. 
3. 



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
